File: first_n_vowelsChallenge.java

// Challenge Link: https://coderhub.sa/challenges/a25be439-0887-4775-9498-ec36eb413888/view?language=Java

public class first_n_vowelsChallenge {
    public static void main(String[] args) {
        System.out.println(first_n_vowels("ShahadAli", 4));
    }

    public static String first_n_vowels(String phrase, int n) {
        String vowels = "";
        for (int i = 0; i < phrase.length(); i++) {
            if (n == 0)
                break;
            if ("aeiou".contains(phrase.toLowerCase().charAt(i) + "")) {
                vowels += phrase.charAt(i);
                n -= 1;
            }
        }
        if (n > 0)
            return "invalid";
        return vowels;
    }
}

File: first_n_vowelsChallenge.py

# Challenge Link: https://coderhub.sa/challenges/a25be439-0887-4775-9498-ec36eb413888/view?language=Python

from typing import List
def first_n_vowels(phrase: str, n: int) -> str:
    # write your code here ^_^
    vols_let = ''
    for char in phrase:
        if str.__contains__("aeiou",char.lower()):
            vols_let+=char
            n-=1
    if n>0:
        print("invalid")
        return ""
    return vols_let
print(first_n_vowels("Riyadh",2))